### DISCUSSION TITLE
What model to use for rotated object detection?

### DISCUSSION THREAD

**Discussion by dreambit** [open-mmlab/mmyolo#970] [General]
Hello, I have a custom dataset of rotated bboxes i made via cvat annotating tool, what model should i use to train and detect rotated bboxes, which one have better accuracy? Thanks

**Comment by SFSimDev** [comment]
I am using rtmdet-r_l_syncbn_fast_2xb4-aug-100e_dota which seems quite good for my needs, but I am only using it as a pretrained model not re-training it.
